protocols/v2/binary-sv2/serde-sv2/src/primitives/sequences/option.rs

```diff
@@ -65,13 +69,28 @@ impl<'s, T: Clone + Serialize + TryFromBSlice<'s>> Serialize for Sv2Option<'s, T
                 seq.end()
             }
             (None, Some(data)) => {
-                let tuple = (data.len() as u8, &data[..]);
-                let mut seq = serializer.serialize_tuple(2)?;
-                seq.serialize_element(&tuple.0)?;
-                seq.serialize_element(tuple.1)?;
-                seq.end()
+                if serializer.is_human_readable() {
+                    let data_ = data.clone();
```

Comment:
u can use masking and do `let data = data.clone();`. I think its more clear

Reply:
you might also be able to remove this `clone` as `serialize_element` takes a reference